graylog2-web-interface/src/components/lookup-tables/caches/CaffeineCacheFieldSet.tsx

```diff
@@ -103,6 +103,17 @@ const CaffeineCacheFieldSet = ({ config }: Props, ref: any) => {
              onChange={handleIgnoreNullChange}
              help="When enabled, null lookup result will be ignored and not cached."
              wrapperClassName="col-md-offset-3 col-md-9" />
+      <TimeUnitInput label="TTL for empty results"
+                     help="Empty results are removed from the cache after the specified time."
```

Comment:
We talk about "null results" and "empty results" interchangeably. Use the same term everywhere to avoid confusion.
Empty results is more accurate I think, since we always return the result data structure (which may contain null field values).